src/main/webapp/user-private-comments.jsp

```diff
@@ -51,13 +51,13 @@
             <%request.setAttribute("clickable", false);%>
             <jsp:include page="task-status-checkmark.jsp"/>
             <p class="inline"><%=task.getDescription()%></p>
-            <form onload="init()" action="/user-profile" method="POST">
-              <input type="hidden" name="taskID" value="<%=task.getTaskID()%>">
+            <form onload="init(box-<%=task.getTaskID()%>, <%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim());" action="/user-profile" method="POST">
```

Comment:
This line and the <input> line below it cause all the issues on the page with loading task. I'm going to do some more specific prints to figure out the exact issues.